pipeline/filters/grep.md

```diff
@@ -14,6 +14,7 @@ The plugin supports the following configuration parameters:
 | :--- | :--- | :--- |
 | Regex | KEY  REGEX | Keep records in which the content of KEY matches the regular expression. |
 | Exclude | KEY REGEX | Exclude records in which the content of KEY matches the regular expression. |
+| Logical_Op| Operation | Specify whether to use logical conjuciton or disjunction. `AND` , `OR` and `legacy` are allowed as an Operation. `legacy` is for backward compatibility. Default is `legacy`.|
```

Comment:
What's the behavior when the value is `legacy`?

Reply:
It is for current behavior to keep backward compatibility.
Current behavior is a buggy but user may expect such behavior.
Could you check from https://github.com/fluent/fluent-bit/pull/6721#issuecomment-1407594534 ?

Reply:
I think we should explain what the legacy mode does, even if it's buggy.

Reply:
My opinion is here. It is hard to explain for me.
https://github.com/fluent/fluent-bit/pull/6721#issuecomment-1409492327

Cc: @patrick-stephens

Reply:
```suggestion
| Logical_Op| Operation | Specify which logical operator to use. `AND` , `OR` and `legacy` are allowed as an Operation. Default is `legacy` for backward compatibility. In `legacy` mode the behaviour is either AND or OR depending whether the `grep` is including (then it's AND) or excluding (then it's OR).|
```